src/components/PricelistForm.js

```diff
@@ -36,5 +48,12 @@ const PricelistForm = (props) => {
   );
 };
 
-const enhance = combine(withHistory, withModulesManager);
-export default enhance(PricelistForm);
+const mapStateToProps = (state) => ({
+  isValid: !!state.medical_pricelist.validationFields?.medicalServices?.isValid || !!state.medical_pricelist.validationFields?.medicalItems?.isValid,
```

Comment:
Can we try to shorten this line? This line could be quite too long in term of potential validations.

Reply:
Done. Moreover, I ordered a little bit the props imports.
`  const {
    readOnly,
    onBack,
    onSave,
    onReset,
    pricelist,
    onChange,
    fetchDetails,
    details,
    isValid,
    clearMedicalPricelists,
  } = props;`

Reply:
good perfect. Thx